source/features/prevent-pr-commit-link-loss.tsx

```diff
@@ -46,7 +46,22 @@ void features.add({
 	screenshot: 'https://user-images.githubusercontent.com/1402241/82131169-93fd5180-97d2-11ea-9695-97051c55091f.gif'
 }, {
 	include: [
-		pageDetect.hasComments
+		pageDetect.hasRichTextEditor
 	],
 	init
+}, {
+	include: [
+		pageDetect.isNewIssue,
+		pageDetect.isCompare
+	],
+	exclude: [
+		() => select.exists('#new_issue .form-actions, #new_pull_request .form-actions')
+	],
+	init: () => {
+		// New issues and Pull Requests are missing the form-actions selector, this avoids having a separate logic just for them.
```

Comment:
You mention that this code is “to avoid separate logic just for them,” but this code is exactly “separate logic just for them,” so let’s integrate this in the original code if necessary

Reply:
Let me explain what I mean by changing the logic.

The entire code  (hiding and showing the bar) is based off the fact that `.form-actions` exists. My idea is to just add the `div` and keep the rest of the code intact.

> so let’s integrate this in the original code if necessary, instead of adding an init function

The **current code** would not make sense to be part of the original code. We would need to check if form actions exists and if its a new pull request/issue. IMHO an init is simpler and cleaner.

> This is still not CSS-only

The only js that could be moved to css is the `formTabContainer?.classList.add('clearfix');`. Do you want to add a file just for that?

@fregante if you feel that there is a better solution then the one I implemented. Then please feel free to close.

Reply:
> is based off the fact that `.form-actions` exists

Is it? Why can't that selector be changed instead? Why add a dummy element instead of just changing a selector?

Reply:
> The only js that could be moved to css is the `formTabContainer?.classList.add('clearfix');`. Do you want to add a file just for that?

You can add the class/rule to `rgh-warning-for-disallow-edits` instead of here

Reply:
I tried to look for another one but nothing looked right. (correct position and not messing anything else up)

I will give it another shot later tonight.

Reply:
I tried again and did not see any selector to append to